Sources/Card/ConfirmPaymentSourceRequest.swift

```diff
@@ -5,50 +5,75 @@ import PaymentsCore
 
 /// Describes request to confirm a payment source (approve an order)
 struct ConfirmPaymentSourceRequest: APIRequest {
-
+    
     private let orderID: String
     private let pathFormat: String = "/v2/checkout/orders/%@/confirm-payment-source"
     private let accessToken: String
     private let jsonEncoder = JSONEncoder()
-
+    
     /// Creates a request to attach a payment source to a specific order.
     /// In order to use this initializer, the `paymentSource` parameter has to
     /// contain the entire dictionary as it exists underneath the `payment_source` key.
     init(
         accessToken: String,
         cardRequest: CardRequest
     ) throws {
+        var confirmPaymentSourceBody = ConfirmPaymentSourceBody()
         var card = cardRequest.card
         if let threeDSecureRequest = cardRequest.threeDSecureRequest {
             let verification = Verification(method: threeDSecureRequest.sca.rawValue)
             card.attributes = Attributes(verification: verification)
+            
+            let applicationContext = ApplicationContext(
+                returnUrl: threeDSecureRequest.returnUrl,
+                cancelUrl: threeDSecureRequest.cancelUrl
```

Comment:
It looks like we need to update our [GitHub docs ](https://github.com/paypal/iOS-SDK/tree/main/docs/Card#5-create-a-request-containing-the-card-payment-details)to include a code snippet on adding these URLs to the `ThreeDSecureRequest`

Reply:
done!